### PR TITLE
add new prerequisites + info on workflows for FATES Galaxy tool tutorial

### PR DESCRIPTION
**Pull request by annefou**
Fix partly #2099 (we will take care of fates-jupyterlab in another pull request) and open this tutorial for review.

### CODE REVIEW

**Comment by bgruening** [comment]
@ColineRoyaux do you have some time review this tutorial? Thanks!

**Comment by ColineRoyaux** [comment]
@bgruening of course ! I'll do it as soon as possible ;)

**Comment by ColineRoyaux** [comment]
Hello @annefou ! Very nice and interesting tutorial ! I really liked the steps 5 and 6 where you show the extraction of the workflow, re-run it and share ! I also liked the "Tip: Finding your tool", it will be very useful for Galaxy newbies !

Overall and as a Climate science novice, It lacked for me mostly context on the data used in this tutorial and explanations on how to intrepret the results properly as a Climatologist would do. However, the integration of such details depends on who this tutorial aims at so it's really not required. 

If you want to use this tutorial only in the Climate and Environment communities it is really good as it is but if you want to use it with students or policy makers I think it is better when a little more details on the scientific background is explained as it helps understanding better and feel more involved in the tutorial :
- You can add in the introduction some context on what contains the two input files at the beginning of the tutorial, maybe where you can find such files, what is exactly the scientific question we are trying to respond to, what models are we going to construct and perhapsdefine some basic technical terms.
- Maybe a little paragraph before the "Hands-on: Creating a new CTSM/FATES-EMERALD case" to explain what we will do in this tool and why this way with these particular parameters
- When inspecting netcdf metadata file, maybe directing trainees to an Index where they can find signification of each acronym used in the file
- The part on Panoply is well explained but a short sentence describing the application and/or a link/citation could be interesting for trainees
- Perhaps a little more details when interpreting the different plots produced
- You can add a few sentences to the conclusion to rapidly sum up what has been done and shown with the models and plots

On the global tutorial there is just some tiny typo to change and a few points that needs to be clarified in my opinion : 
- Step 1 isn't at the same title level as the other steps you need to remove one "#"
- I was advised to supress the "Step 1:", "Step 2:", etc ... from the titles of a tutorial I proposed a few weeks ago so I would advise the same to you 
- "Add a tag to the dataset corresponding to `fates`" it is not very clear which dataset is refered to in this step
- When running the tool CTSM/FATES-EMERALD, maybe precise that the files to check for the datatype are in an output collection as there is also the last output with a .nc extension 
- Precise for Galaxy novices how to add the Extract Dataset tool when editing the workflow and that "netcdf" needs to be selected in the change datatype field

Very interesting tutorial, I currently work on integrating Species Distribution Modeling tools to Galaxy so I'm really happy I learned more about CLM-FATES in your tutorial thank you very much !

**Comment by shiltemann** [comment]
Thanks for your great review @ColineRoyaux! 

@annefou I just pushed some minor formatting tweaks, please check that it all still looks ok to you!

**Comment by annefou** [comment]
Thanks @shiltemann  
and thanks @ColineRoyaux for your thorough review. I will fix the tutorial as soon as possible.

**Comment by annefou** [comment]
Jus to remember: I have now implemented Coline's comment:
- Precise in the edit of the workflow that they have to change the "Atmospheric CO2 molar ratio (by volume) only used when co2_type==constant (umol/mol)" field from 367.0 to 1468.0 to take account of an increased atmospheric Carbon Dioxyde Concentration by a factor of 4. (if I did it properly)

**Comment by annefou** [comment]
@shiltemann it should be now ready for review. @huitang-earth may add some more background in the tutorial part but the slides are ready.

**Comment by shiltemann** [comment]
Thanks @annefou and @huitang-earth !!  I will have a look at this today :)

For the slides: it would be great if you could add speaker notes. Just writing down what you would say about each slide if you were presenting it. This will help other instructors using your slides. :) The syntax to start speaker notes is `???` So something like this:

```
# slide title

bla bla

???
These are the speaker notes

This slide shows ...

Also note that ..
``` 

If you do this, you can also enable your slides for automatic slides-to-video conversion (if you want), see more info here: https://training.galaxyproject.org/training-material/topics/contributing/tutorials/slides-with-video/tutorial.html

File: topics/climate/tutorials/fates/tutorial.md [hunk 1]
```diff
@@ -53,34 +58,34 @@ It will also teach you on how to create Galaxy workflow for your CLM-FATES simul
...
 > ### {% icon comment %} Background
 >
-> FATES is the “Functionally Assembled Terrestrial Ecosystem Simulator”.
+> FATES is the “Functionally Assembled Terrestrial Ecosystem Simulator”, which is a vegetation demographic model ([Fisher et al. 2018](https://onlinelibrary.wiley.com/doi/full/10.1111/gcb.13910)).
 > FATES needs what we call a "Host Land Model" (HLM) to run and in this tutorial
 > we will be using the [Community Land Model](http://www.cesm.ucar.edu/models/clm/)
 > of the [Community Terrestrial Systems Model](https://github.com/ESCOMP/CTSM) (CLM-CTSM).
 > FATES was derived from the CLM Ecosystem Demography model (CLM(ED)), which was documented in
-> {% cite Fisher2015 %}.
+> {% cite Fisher2015 %} and [Koven et al. 2020](https://bg.copernicus.org/articles/17/3017/2020/).
```
Comment (shiltemann) [inline topics/climate/tutorials/fates/tutorial.md:66]: could you add this citation to the `tutorial.bib` file? Then you can use the same syntax as the paper before it `{% cite X %}`

You can use https://www.doi2bib.org/ to convert a doi into a bibtex entry to put in this file

File: topics/climate/tutorials/fates/tutorial.md [hunk 2]
```diff
@@ -53,34 +58,34 @@ It will also teach you on how to create Galaxy workflow for your CLM-FATES simul
 
 > ### {% icon comment %} Background
 >
-> FATES is the “Functionally Assembled Terrestrial Ecosystem Simulator”.
+> FATES is the “Functionally Assembled Terrestrial Ecosystem Simulator”, which is a vegetation demographic model ([Fisher et al. 2018](https://onlinelibrary.wiley.com/doi/full/10.1111/gcb.13910)).
```
Comment (shiltemann) [inline topics/climate/tutorials/fates/tutorial.md:61]: add this one to `tutorial.bib` as well

File: topics/climate/tutorials/fates/tutorial.md [hunk 3]
```diff
@@ -53,34 +58,34 @@ It will also teach you on how to create Galaxy workflow for your CLM-FATES simul
...
 ```
 
-Each sub-folder will then contain all the necessary inputs for running your CLM-FATES case.
-For the purpose of this tutorial, input data for a single point location ALP1 (61.0243N, 8.12343E) has been prepared and is ready to use.
+Each sub-folder will then contain all the necessary inputs for running your CLM-FATES case. For instance, 'atm' contains all the meteorological forcing data for running CLM-FATES. 'lnd' contains the data required to describe surface conditions (e.g., soil depth) for the model. More details about the model inputdata can be found in [CLM and FATES documentation](https://escomp.github.io/ctsm-docs/versions/master/html/tech_note/Ecosystem/CLM50_Tech_Note_Ecosystem.html#model-input-requirements).
+For the purpose of this tutorial, input data for a single point location (ALP1) on the Norwegian alpine tundra ecosystem (Latitude: 61.0243N, Longitude: 8.12343E, Elevation: 1208 m) has been prepared and is ready to use. More details about the sites can be found in [Klanderud et al. (2015)](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0130205) and  [Vigdis et al. (2020)](https://www.pnas.org/content/117/37/22858)
```
Comment (shiltemann) [inline topics/climate/tutorials/fates/tutorial.md:81]: move paper citation to `tutorial.bib`

**Comment by shiltemann** [comment]
Looking good! Could you just rephrase the keypoints to be statements, more like conclusions from the tutorial:

So instead of just "CLM-FATES", make a sentence like "CLM-FATES is a tool that does ..."

(for tutorial and slides)

**Comment by annefou** [comment]
For slides (adding speaker notes), it will take a bit more time but having a video would be really great!!!

**Comment by annefou** [comment]
@shiltemann I finally added the speaker notes!

**Review by shiltemann — APPROVED** [review]
Great work @annefou and @huitang-earth, thanks!!

**Comment by shiltemann** [comment]
oops, I forgot to add a `video: true` metadata to the slides, will do that now. And then it should generate video slides automatically :) (takes a day orso to appear)

**Comment by annefou** [comment]
Awesome! I am really looking forward to it! Many thanks.

**Pull request merged**